src/olympia/abuse/serializers.py

```diff
@@ -98,7 +103,9 @@ def create(self, validated_data):
 
 
 class AddonAbuseReportSerializer(BaseAbuseReportSerializer):
-    addon = serializers.SerializerMethodField()
+    addon = serializers.SerializerMethodField(
+        help_text='The id, slug, or guid of the add-on to report for abuse (required).'
```

Comment:
Is this correct? - `SerializerMethod` is a read-only field so a value can't be provided for it

Reply:
You are right, good catch. I copied the request field and not the response field. Updated